source/_components/image_processing.tensorflow.markdown

```diff
@@ -0,0 +1,161 @@
+---
+layout: page
+title: "TensorFlow"
+description: "Detect and recognize objects with TensorFlow."
+date: 2018-10-24 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tensorflow.png
+ha_category: Image Processing
+featured: false
+ha_release: 0.82
+---
+
+The `tensorflow` image processing platform allows you to detect and recognize objects in a camera image using [TensorFlow](https://www.tensorflow.org/). The state of the entity is the number of objects detected, and recognized objects are listed in the `summary` attribute along with quantity. The `matches` attribute provides the confidence `score` for recognition and the bounding `box` of the object for each detection category.
+
+<p class='note warning'>
+  This package will not yet work on Hass.io installations, but should work on Hassbian.  [See here](https://www.tensorflow.org/install/pip) for system requirements.
+</p>
+
+## {% linkable_title Setup %}
+
+TensorFlow object detection requires some dependencies outside of the core tensorflow package.  These live in the [tensorflow/models](https://github.com/tensorflow/models/tree/master/research/object_detection) GitHub repository.  The one we are interested in for this use case lives in `research/object_detection`.  A general outline of the installation process is as follows:
```

Comment:
I find this instruction a bit confusing myself. Perhaps for clarity something like: 

`
This component requires files to be downloaded, compiled on your computer, and added to the Home-Assistant configuration directory. These steps are can be performed using the sample script at this gist. Alternatively, if you wish to perform the process manually, the process is as follows:
`